pkgs/tools/networking/kail/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "kail-${version}";
+  version = "0.6.0";
+
+  goPackagePath = "github.com/boz/kail";
+
+  src = fetchFromGitHub {
+    owner = "boz";
+    repo = "kail";
+    rev = "v0.6.0";
+    sha256 = "17ybcncdjssil4bn3n2jp1asfcpl8vj560afb2mry9032qrryvx9";
+  };
+
+  # regenerate deps.nix using following steps:
+  #
+  # go get -u github.com/boz/kail
+  # cd $GOPATH/src/github.com/boz/kail
+  # dep init
+  # dep2nix
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "kubernetes log viewer";
+    homepage = "https://github.com/boz/kail";
```

Comment:
Quotes are useless.

Reply:
Even though I was the one to open that PR, there was a significant amount of people not liking it: https://github.com/NixOS/nixpkgs/pull/27809#issuecomment-319494507

Reply:
I don't like too this! I just read your comment and I could also add it's hard for new comers to know this syntax. But I always tries to refer to the manual which doesn't use quote for urls:/